selfdrive/controls/controlsd.py

```diff
@@ -220,9 +220,15 @@ def run(self):
 
 
 def main():
-  config_realtime_process(4, Priority.CTRL_HIGH)
-  controls = Controls()
-  controls.run()
+  try:
+    config_realtime_process(4, Priority.CTRL_HIGH)
+    controls = Controls()
+    controls.run()
+  except KeyboardInterrupt:
+    pass
+  except Exception as e:
+    cloudlog.exception(f"controlsd crashed with exception: {e}")
+    raise
```

Comment:
no need for these, already happens here: https://github.com/commaai/openpilot/blob/60113632b4cbada5b3065540e2c8bc2fd88207e6/system/manager/process.py#L44